containerized_lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+CONTAINER_ENGINE=${CONTAINER_ENGINE:-docker}
+
+${CONTAINER_ENGINE} build --file ./testing.Dockerfile --tag test .
```

Comment:
no need to build the docker each time- lets just use one we will publish ourselves